sql/2_8_3-to-2_9_0_upgrade.sql

```diff
@@ -129,7 +129,7 @@ INSERT INTO layout_options VALUES ('DEM','financial_review','5Stats','Financial
 INSERT INTO layout_options VALUES ('DEM','family_size'     ,'5Stats','Family Size'          , 4, 2,1,20,63,''        ,1,1,'','' ,'Family Size');
 INSERT INTO layout_options VALUES ('DEM','monthly_income'  ,'5Stats','Monthly Income'       , 5, 2,1,20,63,''        ,1,1,'','' ,'Monthly Income');
 INSERT INTO layout_options VALUES ('DEM','homeless'        ,'5Stats','Homeless, etc.'       , 6, 2,1,20,63,''        ,1,1,'','' ,'Homeless or similar?');
-INSERT INTO layout_options VALUES ('DEM','interpretter'    ,'5Stats','Interpreter'          , 7, 2,1,20,63,''        ,1,1,'','' ,'Interpreter needed?');
+INSERT INTO layout_options VALUES ('DEM','interpreter'    ,'5Stats','Interpreter'          , 7, 2,1,20,63,''        ,1,1,'','' ,'Interpreter needed?');
```

Comment:
we are stuck with this misspelling and can't change this one (since it's already populated in database); notably that the label Interpreter (this is what users see) is spelled correctly, at least.